testing_scripts/process_results.py

```diff
@@ -0,0 +1,138 @@
+            new_folder = os.path.join(log_directory,example)
+            # Make new folder (delete if it exists)
+            if os.path.exists(new_folder) == False:
+                os.mkdir(new_folder)
+            # Move the lfind file and results file to folder
+            #os.system(f"mv -f {lfind_file} {new_folder}")
+            try:
+                shutil.copy(src=lfind_file,dst=new_folder)
+            except:
+                print(f"Cannot find the coq file {lfind_file} with lfind in it.")
+            try:
+                if os.path.exists(summary):
+                    #os.system(f"mv -f {summary} {new_folder}")
+                    shutil.copy(src=summary,dst=new_folder)
+                else:
+                    run_log = os.path.join(log_directory,f"lfind_benchmark_{example}")
+                    print(f"Summary log for {lfind_file} not generated. Check {run_log} to see investigate why.")
+            except:
+                print(f"Log not generated for {lfind_file}")
+            # Delete the produced folders if cleaning
+            if clean:
+                os.system(f"rm -R {folder}")
+                og_lfind_folder = os.path.join(parent,example)
+                os.system(f"rm -R {og_lfind_folder}")
+
+def process_results(result_folders,log_directory,clean):
+    content_for_csv = []
+    for file in result_folders:
+        for folder in result_folders[file]:
+            summary = os.path.join(folder,"lfind_summary_log.txt")
+            contents = None
+            example_folder = os.path.basename(folder)
+            example = example_folder[len("_lfind_"):]
+            try:
+                contents = open(summary).read()
+            except:
+                contents = None
+                print(f"--- Error processing log file {summary}")
+                try:
+                    log_file = os.path.join(log_directory,f"lfind_benchmark_log_{example}")
+                    log_contents = open(log_file).read()
+                    if "lemmafinder_example_generation_success" not in log_contents:
+                        print("---- Not able to succcessfully generate examples.")
+                    else:
+                        print("---- Not able to complete running.")
+                except:
+                    print("---- Log not generated properly.")
+            if contents is not None:
+                content_results = parse_contents(contents,summary)
+                content_for_csv.append([
+                    file,
+                    example, 
+                    content_results["Success"], 
+                    content_results["RunTime"], 
+                    content_results["# of Cat 1"], 
+                    content_results["# of Cat 2"], 
+                    content_results["# of Cat 3"], 
+                    content_results["Generalization Count"]
+                ])
+    clean_up_lfind(result_folders,log_directory,clean)
+    return content_for_csv
+            
```

Comment:
We may need more log information that we used in the paper which was done by `process_logs.py`, but for now this looks good to me!

Reply:
Yea I agree, I was just trying to include a place where we could easily log some things and show how to do it and then when we actually want to log specific things, I figured that would probably make sense to add later when we know more explicitly what we are testing for (or maybe we do know and I am just not aware).